packages/TagLabel/TagLabel.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="taglabel">
+    <span class="tag" :class="{partner,community}">
+      <slot />
+    </span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TagLabel',
+  props: {
+    type: { type: String, default: '' }
+  },
+  computed: {
```

Comment:
this computed is not needed.
let's just use the `type` property as a class

Reply:
Fixed